CIP-Fair_Stakepool_Rewards/README.md

```diff
@@ -0,0 +1,195 @@
+---
+CIP: 75
+Title: Fair Stake Pool Rewards
+Status: Proposed
+Category: Reward-Sharing Scheme
+Authors:
+    - Tobias Fancee <tobiasfancee@gmail.com>
+Implementors:
+    - Input Output Global
+    - Cardano Foundation
+    - Emurgo
+Discussions:
+    - https://forum.cardano.org/t/cip-fair-stakepool-rewards/109368
+Created: 2022-10-21
+License: CC-BY-4.0
+---
+
+# Abstract
```

Comment:
```suggestion
# CIP-0013: Cardano URI Scheme

## Abstract
```
... and likewise demoting the headers all the way down to a more web-friendly outline (in which pages only have one H1 and major sections are H2's) ...